README.md

```diff
@@ -1,3 +1,8 @@
+# Notebook Renderers for Visual Studio Code
+
+This extension provides....
+Place an image of Ploty/vega
```

Comment:
Does this mean something?

Reply:
Its basically a todo item.
I'll need some description and image for the readme page (splash page for extension on marketplace).
Got a GitHub issue tracking this, will chase this up with PMs.